compositor_render/src/wgpu/shader/pipeline.rs

```diff
@@ -72,33 +75,45 @@ impl Pipeline {
             },
             vertex: wgpu::VertexState {
                 buffers: &[Vertex::LAYOUT],
-                module: &shader_module,
+                module: shader_module,
                 entry_point: super::VERTEX_ENTRYPOINT_NAME,
             },
             fragment: Some(wgpu::FragmentState {
-                module: &shader_module,
+                module: shader_module,
                 entry_point: super::FRAGMENT_ENTRYPOINT_NAME,
                 targets: &[Some(wgpu::ColorTargetState {
                     format: wgpu::TextureFormat::Rgba8Unorm,
                     write_mask: wgpu::ColorWrites::all(),
                     blend: Some(wgpu::BlendState::ALPHA_BLENDING),
                 })],
             }),
-            layout: Some(&pipeline_layout),
+            layout: Some(pipeline_layout),
             multisample: wgpu::MultisampleState {
                 count: 1,
                 mask: !0,
                 alpha_to_coverage_enabled: false,
             },
             multiview: None,
-        });
+        })
+    }
 
-        Self {
-            pipeline,
-            sampler,
-            surfaces,
-            textures_bgl,
-        }
+    pub fn create_texture_bind_group_layout(
+        device: &wgpu::Device,
+        texture_count: u32,
+    ) -> wgpu::BindGroupLayout {
+        device.create_bind_group_layout(&wgpu::BindGroupLayoutDescriptor {
+            label: Some("shader transformation textures bgl"),
```

Comment:
`WgpuShader` uses that. It's not specific for `shader transformation`. This may produce misleading logs/errors, since this doesn't have to be `shader transformation` stuff.

Reply:
Not added in this PR + currently that code is used for everything so probably the only correct label here would be None :D